package.json

```diff
@@ -30,7 +30,7 @@
   },
   "homepage": "https://github.com/Netflix/eslint-config-netflix",
   "peerDependencies": {
-    "eslint": "^8.57.0"
+    "@eslint/js": "^8.x"
```

Comment:
This is really all we depend on. We want this plugin to attempt to be as flexible as possible to prevent version conflicts.

Reply:
I think we would require `8.21` of ESLint which is when the flat config format was introduced (still trying to find some reference about that...)

Reply:
* also ESLint/js may not have the same version alignment ...

Reply:
I can just go back to `^8.57.0` as well.

Reply:
> also ESLint/js may not have the same version alignment ...

Reply:
here's the release notes announcing `8.21` is the version we care about: https://eslint.org/blog/2022/08/eslint-v8.21.0-released/